package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "async": "0.2.6",
     "atom-keymap": "^6.1.1",
+    "aws-sdk": "^2.2.18",
```

Comment:
Given that we only need for this CI, should we install it in `cibuild` (or some such) instead? @atom/feedback